docs/source/basic_usage.rst

```diff
@@ -1,66 +1,55 @@
 Basic Usage
 ===========
 
-Let's say you have an app **family** with a model like this:
+Let's say you have an app **shop** with a model like this:
 
-File: model.py ::
+File: **models.py** ::
 
-    class Kid(models.Model):
+    class Customer(models.Model):
         """
-        Model class Kid of family app
+        Model class Customer of shop app
         """
         happy = models.BooleanField()
```

Comment:
Although I agree with you I feel that changing this will require a lot of effort on reviewing all the docs once again because it's 90% based in this `Customer` user. I'll create an issue to document this so people can help us to improve our documentation just to unblock us from not releasing `1.0.0`. Issue #15 now covers this discussion.